priv/code_navigation/src/diagnostics_xref.erl

```diff
@@ -0,0 +1,6 @@
+-module(diagnostics_xref).
+
+-export([ main/0 ]).
+
+main() ->
+  lists:map(1, 2, 3).
```

Comment:
Should there be a function call that is valid as well in this example module?

Reply:
Good point. Added calls to both local and non-local, existing and non-existing functions. One is also picked by the compiler.